### PR TITLE
[bitnami/zookeeper] Configure headless service name

### PR DESCRIPTION
**Pull request by fabien-marchand**
A new configuration parameter `service.headless.name` now allows to override the default name given to the headless service.

Signed-off-by: Fabien Marchand <fabien.marchand@intersec.com>

### Applicable issues

  - fixes #

### Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [ ] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [ ] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/zookeeper/Chart.yaml [hunk 1]
```diff
@@ -21,4 +21,4 @@ name: zookeeper
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/zookeeper
   - https://zookeeper.apache.org/
-version: 11.0.2
+version: 11.0.3
```
Comment (jotamartos) [inline bitnami/zookeeper/Chart.yaml:24]: As you are adding a new parameter, please bump the version to a new minor (`11.1.0`)

**Review by jotamartos — CHANGES_REQUESTED** [review]
Thank you for your  contribution. However, I'd like you to take a look at how other charts were configured to apply the same changes here. You can take a look at RabbitMQ as an example 

https://github.com/bitnami/charts/blob/main/bitnami/rabbitmq/values.yaml#L80

**Comment by fabien-marchand** [comment]
> Thank you for your contribution. However, I'd like you to take a look at how other charts were configured to apply the same changes here. You can take a look at RabbitMQ as an example
> 
> https://github.com/bitnami/charts/blob/main/bitnami/rabbitmq/values.yaml#L80

Thanks for the review. I re-pushed but did not exactly as it is done in "rabbitmq" (the service name is not a full override as you have a `%s-%s` constraint, which I d

**Comment by jotamartos** [comment]
Please resolve the conflict so all tests pass. Thanks!

**Comment by jotamartos** [comment]
Hi @fabien-marchand,

Did you take a look at the comment I posted above? 

https://github.com/bitnami/charts/pull/14144#pullrequestreview-1242134542

I suggested you use the "servicenameOverride" variable as others solution use.

**Comment by fabien-marchand** [comment]
> servicenameOverride

Hi @jotamartos ,
Yes, sorry I did not undertstand you were also talking about setting the exact same name for the option. I just repushed that way, though the name `service.headless.servicenameOverride` seems a bit redundant to me and not really exact as the name is not fully overriden.

**Comment by jotamartos** [comment]
Hi again @fabien-marchand,

Sorry if the info was not clear enough. I asked you to implement these changes in the bitnami/zookeeper chart. Can you update the PR to be like this other one?

https://github.com/bitnami/charts/commit/e963f17318085b18eb87f88d44415da7ecd4f3d0

Thanks

**Comment by jotamartos** [comment]
>though the name service.headless.servicenameOverride seems a bit redundant to me and not really exact as the name is not fully overriden.

I see your point of view but I'd like to keep it in sync with other charts we have in which we also use that variable to partially change the name of the service.

**Review by jotamartos — APPROVED** [review]

**Pull request merged**